src/common_runtime/interface/event_looper.h

```diff
@@ -0,0 +1,42 @@
+# pragma once 
+
+/// error code
+#define CR_OK 0
+#define CR_ERR -1
+
+/// mask
+#define CR_NONE 0
+#define CR_READABLE 1
+#define CR_WRITABLE 2
+
+/// events
+#define CR_FILE_EVENTS 1
+#define CR_TIME_EVENTS 2
+#define CR_ALL_EVENTS (CR_FILE_EVENTS|CR_TIME_EVENTS)
+#define CR_DONT_WAIT 4
+
+/// timer return
+#define CR_NOMORE -1
+#define CR_DELETED_EVENT_ID -1
+
+struct CrEventLoop;
+
+/* callback types */
```

Comment:
We've moved away from using `/* */`, so this should be `//`.

`///` should only be used for documentation that should be parsed by doxygen.